sdks/go/examples/stringsplit/offsetrange/offsetrange.go

```diff
@@ -89,21 +89,23 @@ func (tracker *Tracker) GetError() error {
 }
 
 // TrySplit splits at the nearest integer greater than the given fraction of the remainder.
-func (tracker *Tracker) TrySplit(fraction float64) (interface{}, error) {
+func (tracker *Tracker) TrySplit(fraction float64) (interface{}, interface{}, error) {
 	if tracker.Stopped || tracker.IsDone() {
-		return nil, nil
+		return tracker.Rest, nil, nil
 	}
-	if fraction < 0 || fraction > 1 {
-		return nil, errors.New("fraction must be in range [0, 1]")
+	if fraction < 0 {
+		fraction = 0
```

Comment:
Might be worth documenting this behavior in the comment.

Reply:
Done.